bastionzero/environment/resource_environment_test.go

```diff
@@ -93,6 +93,60 @@ func TestAccEnvironment_Disappears(t *testing.T) {
 	})
 }
 
+func TestAccEnvironment_Name(t *testing.T) {
+	ctx := context.Background()
+	rDesc := acctest.RandomName()
+	name1 := "development_environment"
+	name2 := "development"
```

Comment:
Use `acctest.RandomName()` because sweeping logic of leftover resources works by having correct prefix in named BastionZero API resources.

Example:

https://github.com/bastionzero/terraform-provider-bastionzero/blob/ef3edad008c362b536eb370f4130a2ec4675224c/bastionzero/policy/sessionrecording/resource_sessionrecordingpolicy_test.go#L95-L96